laravel-app/resources/views/profile/show.blade.php

```diff
@@ -10,10 +10,29 @@
 				<img class="img-fluid w-50" src="{{ asset('assets/images/microblog-logo.png') }}" alt="Microblog Logo">
 			</div>
 			<div class="col-6 pt-1">
-				<div class="row">
+				<div class="row justify-content-between">
 					<div class="col">
 						<label>Name</label>
-						<p class="fw-bold">{{ $user->user_information->first_name . ' ' . $user->user_information->middle_name . ' ' . $user->user_information->last_name}} </p>
+						<p class="fw-bold">{{
+							$user->user_information->first_name . ' ' . 
+							$user->user_information->middle_name . ' ' . 
+							$user->user_information->last_name
+						}}</p>
+					</div>
+					<div class="col align-self-center">
+						@if(auth()->user()->isNot($user))
+							@if(auth()->user()->following->contains($user))
```

Comment:
maybe can you create a function on this like 
`$user->isFollows($user)`

Reply:
done sir, added a new function in User model to check if the user is a following with isFollowing method